pep-0517.txt

```diff
@@ -218,6 +241,14 @@ metadata. The directory passed in by the build frontend MUST be
 identical to the directory created by ``get_wheel_metadata``,
 including any unrecognized files it created.
 
+This method only places the *contents* of the wheel file, without actually
+creating the final artifact. This is to allow flexibility in how the result is
+going to be used. A front end like pip, which is only going to immediately
+install this wheel can speed up the installation process by skipping a round
+trip through zip. However a command like a hypothetical ``twine wheel`` could
```

Comment:
> A front end like pip, which is only going to immediately install this wheel can speed up the installation process by skipping a round trip through zip.

Is this true for `pip`, though? Won't pip want to drop the built wheel in the wheel cache?

BTW I have this idea of pip eventually switching to having a wheel cache of unpacked wheels and just symlinking or hard-linking them inside the virtualenv, but this is a subject for another space...